rented-properties/src/main/java/org/egov/cpt/service/AccountStatementExcelGenerationService.java

```diff
@@ -0,0 +1,166 @@
+package org.egov.cpt.service;
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.time.Instant;
+import java.time.ZoneId;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import org.apache.poi.ss.usermodel.Cell;
+import org.apache.poi.ss.usermodel.CellStyle;
+import org.apache.poi.ss.usermodel.Font;
+import org.apache.poi.ss.usermodel.IndexedColors;
+import org.apache.poi.ss.usermodel.Row;
+import org.apache.poi.ss.usermodel.Sheet;
+import org.apache.poi.ss.usermodel.Workbook;
+import org.apache.poi.xssf.usermodel.XSSFWorkbook;
+import org.egov.common.contract.request.RequestInfo;
+import org.egov.cpt.models.AccountStatementCriteria;
+import org.egov.cpt.models.Property;
+import org.egov.cpt.models.PropertyCriteria;
+import org.egov.cpt.models.RentAccountStatement;
+import org.egov.cpt.repository.PropertyRepository;
+import org.egov.cpt.util.FileStoreUtils;
+import org.egov.cpt.web.contracts.AccountStatementResponse;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+@Service
+public class AccountStatementExcelGenerationService {
+
+	private PropertyRepository propertyRepository;
+	private PropertyService propertyService;
+	private FileStoreUtils fileStoreUtils;
+
+	private static final DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern("dd/MM/yyyy");
+	private static String[] columns = { "Date", "Amount", "Type", "Interest Due", "Principal Due", "Total Due",
+			"Account Balance" };
+	private static String[] propertyColumns = { "Name", "Date of Allotment As Per Lease Deed", "Plot No.", "Area",
+			"Rent", "Security Advance Taken By o/o BDPO U.T.Interest",
+			"Yr. Rent (increase as per Clause 4 of Lease Deed)", "Montly Rent", "Interest" };
+
+	@Autowired
+	public AccountStatementExcelGenerationService(PropertyRepository propertyRepository,
+			PropertyService propertyService, FileStoreUtils fileStoreUtils) {
+		this.propertyRepository = propertyRepository;
+		this.propertyService = propertyService;
+		this.fileStoreUtils = fileStoreUtils;
+	}
+
+	public String generageAccountStatementExcel(AccountStatementCriteria accountStatementCriteria,
```

Comment:
Change the return type to `List<HashMap<String, String>>` exactly what will be returned by `fileStoreUtils.fetchFileStoreId`

Reply:
return type changed to list